### PR TITLE
[Docs] Refine the documentation

### PR DESCRIPTION
**Pull request by Xiangxu-0103**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/advanced_guides/customize_dataset.md [hunk 1]
```diff
@@ -20,35 +20,35 @@ Currently, we only support '.bin' format point cloud for training and inference.
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_dataset.md:20]: Currently, we only support '.bin' format -> Currently, we only support `.bin` format

File: docs/en/advanced_guides/customize_dataset.md [hunk 2]
```diff
@@ -20,35 +20,35 @@ Currently, we only support '.bin' format point cloud for training and inference.
 
 1. Convert pcd to bin: https://github.com/DanielPollithy/pypcd
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_dataset.md:21]: Conver `.pcd` to `.bin`

File: docs/en/advanced_guides/customize_dataset.md [hunk 3]
```diff
@@ -20,35 +20,35 @@ Currently, we only support '.bin' format point cloud for training and inference.
...
+  points = np.zeros([pcd_data.width, 4], dtype=np.float32)
+  points[:, 0] = pcd_data.pc_data['x'].copy()
+  points[:, 1] = pcd_data.pc_data['y'].copy()
+  points[:, 2] = pcd_data.pc_data['z'].copy()
+  points[:, 3] = pcd_data.pc_data['intensity'].copy().astype(np.float32)
+  with open('point_cloud_data.bin', 'wb') as f:
+      f.write(points.tobytes())
+  ```
 
 2. Convert las to bin: The common conversion path is las -> pcd -> bin, and the conversion from las -> pcd can be achieved through [this tool](https://github.com/Hitachi-Automotive-And-Industry-Lab/semantic-segmentation-editor).
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_dataset.md:45]: Convert `.las` to `.bin`

File: docs/en/advanced_guides/customize_dataset.md [hunk 4]
```diff
@@ -20,35 +20,35 @@ Currently, we only support '.bin' format point cloud for training and inference.
...
+- You can install `pypcd` with the following command:
 
-```bash
-pip install git+https://github.com/DanielPollithy/pypcd.git
-```
+  ```bash
+  pip install git+https://github.com/DanielPollithy/pypcd.git
+  ```
 
-- You can use the following command to read the pcd file and convert it to bin format and save it:
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_dataset.md:29]: `.pcd`  xxxx `.bin`

File: docs/en/advanced_guides/customize_runtime.md [hunk 5]
```diff
@@ -1,51 +1,52 @@
-# Customize Runtime Settings
+# 自定义运行时配置
 
-## Customize optimization settings
+## 自定义优化器设置
 
-Optimization related configuration is now all managed by `optim_wrapper` which usually has three fields: `optimizer`, `paramwise_cfg`, `clip_grad`. Please refer to [OptimWrapper](https://mmengine.readthedocs.io/en/latest/tutorials/optim_wrapper.md) for more detail. See the example below, where `Adamw` is used as an `optimizer`, the learning rate of the backbone is reduced by a factor of 10, and gradient clipping is added.
+优化器相关的配置是由 `optim_wrapper` 管理的，其通常有三个字段：`optimizer`，`paramwise_cfg`，`clip_grad`。请参考 [OptimWrapper](https://mmengine.readthedocs.io/zh_CN/latest/tutorials/optim_wrapper.html) 了解更多细节。如下所示，使用 `AdamW` 作为`优化器`，骨干网络的学习率降低 10 倍，并添加了梯度裁剪。
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:5]: 更多细节请参考 xxxxx

File: docs/en/advanced_guides/customize_runtime.md [hunk 6]
```diff
@@ -1,51 +1,52 @@
...
+    # 梯度裁剪
     clip_grad=dict(max_norm=0.01, norm_type=2))
 ```
 
-### Customize optimizer supported by PyTorch
+### 自定义 PyTorch 支持的优化器
 
-We already support to use all the optimizers implemented by PyTorch, and the only modification is to change the `optimizer` field in `optim_wrapper` field of config files. For example, if you want to use `ADAM` (note that the performance could drop a lot), the modification could be as the following.
+我们已经支持使用所有 PyTorch 实现的优化器，且唯一需要修改的地方就是改变配置文件中的 `optim_wrapper` 字段中的 `optimizer` 字段。
+举个例子，如果您想使用 `ADAM`（注意到这样可能会使性能大幅下降），您可以这样修改：
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:32]: 例如 （用词要更正式一点）

File: docs/en/advanced_guides/customize_runtime.md [hunk 7]
```diff
@@ -59,52 +60,53 @@ class MyOptimizer(Optimizer):
 
 ```
 
-#### 2. Add the optimizer to registry
+#### 2. 将优化器添加到注册器
 
-To find the above module defined above, this module should be imported into the main namespace at first. There are two options to achieve it.
+为了找到上述定义的优化器模块，该模块首先需要被引入主命名空间。有两种方法实现之：
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:65]: 有两种实现方法

File: docs/en/advanced_guides/customize_runtime.md [hunk 8]
```diff
@@ -59,52 +60,53 @@ class MyOptimizer(Optimizer):
...
 
-- Modify `mmdet3d/engine/optimizers/__init__.py` to import it.
+- 新建 `mmdet3d/engine/optimizers/__init__.py` 文件用于引入。
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:67]: 新建 -> 修改

Comment (ZCMax) [reply docs/en/advanced_guides/customize_runtime.md:67]: 修改 xxxx 导入该模块

File: docs/en/advanced_guides/customize_runtime.md [hunk 9]
```diff
@@ -59,52 +60,53 @@ class MyOptimizer(Optimizer):
...
 
-  The newly defined module should be imported in `mmdet3d/engine/optimizers/__init__.py` so that the registry will find the new module and add it:
+  新定义的模块应该在 `mmdet3d/engine/optimizers/__init__.py` 中被引入，使得注册器可以找到新模块并注册之：
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:69]: 引入 -> 导入

Comment (ZCMax) [reply docs/en/advanced_guides/customize_runtime.md:69]: 从而被找到并且被添加到注册器中

File: docs/en/advanced_guides/customize_runtime.md [hunk 10]
```diff
@@ -59,52 +60,53 @@ class MyOptimizer(Optimizer):
...
 
-```python
-from .my_optimizer import MyOptimizer
-```
+  ```python
+  from .my_optimizer import MyOptimizer
+  ```
 
-- Use `custom_imports` in the config to manually import it
+- 在配置中使用 `custom_imports` 来人工引入新优化器：
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:75]: 引入 -> 导入

File: docs/en/advanced_guides/customize_runtime.md [hunk 11]
```diff
@@ -59,52 +60,53 @@ class MyOptimizer(Optimizer):
...
 
-```python
-custom_imports = dict(imports=['mmdet3d.engine.optimizers.my_optimizer'], allow_failed_imports=False)
-```
+  ```python
+  custom_imports = dict(imports=['mmdet3d.core.optimizer.my_optimizer'], allow_failed_imports=False)
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:78]: engine.optimizer

File: docs/en/advanced_guides/customize_runtime.md [hunk 12]
```diff
@@ -59,52 +60,53 @@ class MyOptimizer(Optimizer):
...
+为了使用您自己的优化器，该字段可以改为：
 
 ```python
 optim_wrapper = dict(
     type='OptimWrapper',
     optimizer=dict(type='MyOptimizer', a=a_value, b=b_value, c=c_value))
 ```
 
-### Customize optimizer wrapper constructor
+### 自定义优化器的构造器
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:106]: 自定义优化器封装构造器

File: docs/en/advanced_guides/customize_runtime.md [hunk 13]
```diff
@@ -59,52 +60,53 @@ class MyOptimizer(Optimizer):
...
 
-Some models may have some parameter-specific settings for optimization, e.g. weight decay for BatchNorm layers.
-The users can do those fine-grained parameter tuning through customizing optimizer wrapper constructor.
+部分模型可能会拥有一些参数专属的优化器设置，比如 BatchNorm 层的权重衰减 (weight decay)。
+用户可以通过自定义优化器的构造器来对那些细粒度的参数进行调优。
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:109]: 自定义优化器封装构造器

File: docs/en/advanced_guides/customize_runtime.md [hunk 14]
```diff
@@ -126,32 +128,34 @@ class MyOptimizerWrapperConstructor(DefaultOptimWrapperConstructor):
 
 ```
 
-The default optimizer wrapper constructor is implemented [here](https://github.com/open-mmlab/mmengine/blob/main/mmengine/optim/optimizer/default_constructor.py#L18), which could also serve as a template for the new optimizer wrapper constructor.
+默认优化器构造器在[这里](https://github.com/open-mmlab/mmengine/blob/main/mmengine/optim/optimizer/default_constructor.py#L18)实现。这部分代码也可以用作新优化器构造器的模版。
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:131]: 优化器封装构造器 （可以再 check 一下别的地方）

File: docs/en/advanced_guides/customize_runtime.md [hunk 15]
```diff
@@ -126,32 +128,34 @@ class MyOptimizerWrapperConstructor(DefaultOptimWrapperConstructor):
...
+  如果您的配置继承了一个已经设置了 `optim_wrapper` 的基础配置，那么您可能需要 `_delete_=True` 字段来覆盖基础配置中无用的设置。详见配置文件的[说明文档](https://mmdetection3d.readthedocs.io/en/latest/tutorials/config.html)。
 
-- __Use momentum schedule to accelerate model convergence__:
-  We support momentum scheduler to modify model's momentum according to learning rate, which could make the model converge in a faster way.
-  Momentum scheduler is usually used with LR scheduler, for example, the following config is used in [3D detection](https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/configs/_base_/schedules/cyclic_20e.py) to accelerate convergence.
-  For more details, please refer to the implementation of [CosineAnnealingLR](https://github.com/open-mmlab/mmengine/blob/main/mmengine/optim/scheduler/lr_scheduler.py#L43) and [CosineAnnealingMomentum](https://github.com/open-mmlab/mmengine/blob/main/mmengine/optim/scheduler/momentum_scheduler.py#L71).
+- __使用动量规划器 (momentum scheduler) 来加速模型收敛__：
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:148]: 动量调度器（ check 其他地方

File: docs/en/advanced_guides/customize_runtime.md [hunk 16]
```diff
@@ -126,32 +128,34 @@ class MyOptimizerWrapperConstructor(DefaultOptimWrapperConstructor):
...
+  动量规划器通常和学习率规划器一起使用，比如说，如下配置文件在 [3D 检测](https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/configs/_base_/schedules/cyclic_20e.py)中被用于加速模型收敛。
+  更多细节详见 [CosineAnnealingLR](https://github.com/open-mmlab/mmengine/blob/main/mmengine/optim/scheduler/lr_scheduler.py#L43) 和 [CosineAnnealingMomentum](https://github.com/open-mmlab/mmengine/blob/main/mmengine/optim/scheduler/momentum_scheduler.py#L71) 的实现。
 
   ```python
   param_scheduler = [
-      # learning rate scheduler
-      # During the first 8 epochs, learning rate increases from 0 to lr * 10
-      # during the next 12 epochs, learning rate decreases from lr * 10 to lr * 1e-4
+      # 学习率调整策略
+      #在前 8 轮，学习率从 0 升到 lr * 10
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:157]: 空格

Comment (ZCMax) [reply docs/en/advanced_guides/customize_runtime.md:157]: 前 8 个 epoch ( epoch 保留英文

File: docs/en/advanced_guides/customize_runtime.md [hunk 17]
```diff
@@ -168,9 +172,9 @@ Tricks not implemented by the optimizer should be implemented through optimizer
           end=20,
           by_epoch=True,
           convert_to_iter_based=True),
-      # momentum scheduler
-      # During the first 8 epochs, momentum increases from 0 to 0.85 / 0.95
-      # during the next 12 epochs, momentum increases from 0.85 / 0.95 to 1
+      # 动量调整策略
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:175]: 动量调度器

File: docs/en/advanced_guides/customize_runtime.md [hunk 18]
```diff
@@ -190,12 +194,12 @@ Tricks not implemented by the optimizer should be implemented through optimizer
   ]
   ```
 
-## Customize training schedules
+## 自定义训练规程
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:197]: 自定义训练调度

File: docs/en/advanced_guides/customize_runtime.md [hunk 19]
```diff
@@ -190,12 +194,12 @@ Tricks not implemented by the optimizer should be implemented through optimizer
...
 
-By default we use step learning rate with 1x schedule, this calls [`MultiStepLR`](https://github.com/open-mmlab/mmengine/blob/main/mmengine/optim/scheduler/lr_scheduler.py#L139) in MMEngine.
-We support many other learning rate schedule [here](https://github.com/open-mmlab/mmengine/blob/main/mmengine/optim/scheduler/lr_scheduler.py), such as `CosineAnnealingLR` and `PolyLR` schedules. Here are some examples
+默认情况，我们使用阶梯式学习率衰减的 1 倍训练规程。这会调用 MMEngine 中的 [`MultiStepLR`](https://github.com/open-mmlab/mmengine/blob/main/mmengine/optim/scheduler/lr_scheduler.py#L139)。
+我们在[这里](https://github.com/open-mmlab/mmengine/blob/main/mmengine/optim/scheduler/lr_scheduler.py)支持很多其他学习率规划方案，比如`余弦退火`和`多项式衰减`规程。下面是一些样例：
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:200]: 支持了

Comment (ZCMax) [reply docs/en/advanced_guides/customize_runtime.md:200]: 学习率调度策略    规程-> 调度

File: docs/en/advanced_guides/customize_runtime.md [hunk 20]
```diff
@@ -190,12 +194,12 @@ Tricks not implemented by the optimizer should be implemented through optimizer
...
 
-- Poly schedule:
+- 多项式衰减规程:
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:202]: 规程 -> 调度

File: docs/en/advanced_guides/customize_runtime.md [hunk 21]
```diff
@@ -221,20 +225,20 @@ We support many other learning rate schedule [here](https://github.com/open-mmla
           by_epoch=True)]
   ```
 
-## Customize train loop
+## 自定义工作流
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:228]: 自定义训练循环控制器

File: docs/en/advanced_guides/customize_runtime.md [hunk 22]
```diff
@@ -221,20 +225,20 @@ We support many other learning rate schedule [here](https://github.com/open-mmla
...
 
-By default, `EpochBasedTrainLoop` is used in `train_cfg` and validation is done after every train epoch, as follows.
+我们默认在 `train_cfg` 中使用 `EpochBasedTrainLoop`，并在每一个训练周期完全后执行一次验证，如下所示：
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:230]: 每一个训练 epoch 完成后进行一次验证

File: docs/en/advanced_guides/customize_runtime.md [hunk 23]
```diff
@@ -221,20 +225,20 @@ We support many other learning rate schedule [here](https://github.com/open-mmla
...
 ```
 
-Actually, both [`IterBasedTrainLoop`](https://github.com/open-mmlab/mmengine/blob/main/mmengine/runner/loops.py#L183%5D) and [`EpochBasedTrainLoop`](https://github.com/open-mmlab/mmengine/blob/main/mmengine/runner/loops.py#L18) support dynamical interval, see the following example.
+事实上，[`IterBasedTrainLoop`](https://github.com/open-mmlab/mmengine/blob/main/mmengine/runner/loops.py#L183%5D) 和 [`EpochBasedTrainLoop`](https://github.com/open-mmlab/mmengine/blob/main/mmengine/runner/loops.py#L18) 都支持动态间隔验证，如下所示：
 
 ```python
-# Before 365001th iteration, we do evaluation every 5000 iterations.
-# After 365000th iteration, we do evaluation every 368750 iterations,
-# which means that we do evaluation at the end of training.
+# 在第 365001 个迭代之前，我们每隔 5000 个迭代验证一次
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:239]: 个 -> 次

File: docs/en/advanced_guides/customize_runtime.md [hunk 24]
```diff
@@ -290,63 +294,63 @@ class MyHook(Hook):
...
 ```
 
-Depending on the functionality of the hook, users need to specify what the hook will do at each stage of the training in `before_run`, `after_run`, `before_train`, `after_train` , `before_train_epoch`, `after_train_epoch`, `before_train_iter`, and `after_train_iter`.  There are more points where hooks can be inserted, refer to [base hook class](https://github.com/open-mmlab/mmengine/blob/main/mmengine/hooks/hook.py#L9) for more detail.
+取决于钩子的功能，用户需要指定钩子在每个训练阶段时的行为，具体包括如下阶段：`before_run`，`after_run`，`before_train`，`after_train`，`before_train_epoch`，`after_train_epoch`，`before_train_iter`，和 `after_train_iter`。有更多的位点可以插入钩子，详情可参考 [base hook class](https://github.com/open-mmlab/mmengine/blob/main/mmengine/hooks/hook.py#L9)。
 
-#### 2. Register the new hook
+#### 2. 注册新钩子
 
-Then we need to make `MyHook` imported. Assuming the file is in `mmdet3d/engine/hooks/my_hook.py` there are two ways to do that:
+接下来我们需要引入 `MyHook`。假设新钩子位于文件 `mmdet3d/engine/hooks/my_hook.py` 中，有两种方法可以实现之：
```
Comment (ZCMax) [inline docs/en/advanced_guides/customize_runtime.md:301]: 引入 -> 导入

File: docs/en/advanced_guides/datasets/kitti_det.md [hunk 25]
```diff
@@ -182,30 +183,30 @@ An example to test PointPillars on KITTI with 8 GPUs and generate a submission t
...
-data_root = 'data/kitti/'
-test_evaluator = dict(
-    type='KittiMetric',
-    ann_file=data_root + 'kitti_infos_test.pkl',
-    metric='bbox',
-    pklfile_prefix='results/kitti-3class/kitti_results',
-    submission_prefix='results/kitti-3class/kitti_results')
-```
+  ```python
+  data_root = 'data/kitti/'
```
Comment (ZCMax) [inline docs/en/advanced_guides/datasets/kitti_det.md:187]:
```python
data_root = 'data/kitti/'
test_dataloader = dict(
    dataset=dict(ann_file='kitti_infos_test.pkl', load_eval_anns=False, data_prefix=dict(img='testing/image_2')))
test_evaluator = dict(
    ann_file=data_root + 'kitti_infos_test.pkl',
    format_only=True,
    pklfile_prefix='results/kitti-mono3d/kitti_results',
    submission_prefix='results/kitti-mono3d/kitti_results'
    )
```

Comment (ZCMax) [reply docs/en/advanced_guides/datasets/kitti_det.md:187]: Please also update the corresponding Chinese doc

Comment (ZCMax) [reply docs/en/advanced_guides/datasets/kitti_det.md:187]: We updated the KITTI metric in https://github.com/open-mmlab/mmdetection3d/pull/2005

File: docs/en/advanced_guides/datasets/kitti_det.md [hunk 26]
```diff
@@ -182,30 +183,30 @@ An example to test PointPillars on KITTI with 8 GPUs and generate a submission t
 
 - First, you need to modify the `test_evaluator` dict in your config file to add `pklfile_prefix` and `submission_prefix`, just like:
```
Comment (ZCMax) [inline docs/en/advanced_guides/datasets/kitti_det.md:184]: First, you need to modify the `test_dataloader` and `test_evaluator` dict in your config file,  just like:

File: docs/en/advanced_guides/datasets/kitti_det.md [hunk 27]
```diff
@@ -182,30 +183,30 @@ An example to test PointPillars on KITTI with 8 GPUs and generate a submission t
...
+      pklfile_prefix='results/kitti-3class/kitti_results',
+      submission_prefix='results/kitti-3class/kitti_results')
+  ```
 
 - And then, you can run the test script.
 
-```shell
-mkdir -p results/kitti-3class
+  ```shell
+  mkdir -p results/kitti-3class
```
Comment (ZCMax) [inline docs/en/advanced_guides/datasets/kitti_det.md:199]: remove `mkdir -p results/kitti-3class`

File: docs/en/advanced_guides/datasets/kitti_det.md [hunk 28]
```diff
@@ -182,30 +183,30 @@ An example to test PointPillars on KITTI with 8 GPUs and generate a submission t
...
 
-./tools/dist_test.sh configs/pointpillars/configs/pointpillars/pointpillars_hv_secfpn_8xb6-160e_kitti-3d-3class.py work_dirs/pointpillars_hv_secfpn_8xb6-160e_kitti-3d-3class/latest.pth 8
-```
+  ./tools/dist_test.sh configs/pointpillars/pointpillars_hv_secfpn_8xb6-160e_kitti-3d-3class.py work_dirs/pointpillars_hv_secfpn_8xb6-160e_kitti-3d-3class/latest.pth 8
+  ```
 
 - Or you can use `--cfg-options "test_evaluator.pklfile_prefix=results/kitti-3class/kitti_results" "test_evaluator.submission_prefix=results/kitti-3class/kitti_results"` after the test command, and run test script directly.
```
Comment (ZCMax) [inline docs/en/advanced_guides/datasets/kitti_det.md:204]: Remove `Or you can use `--cfg-options "test_evaluator.pklfile_prefix=results/kitti-3class/kitti_results" "test_evaluator.submission_prefix=results/kitti-3class/kitti_results"` after the test command, and run test script directly.`

File: docs/en/advanced_guides/datasets/kitti_det.md [hunk 29]
```diff
@@ -182,30 +183,30 @@ An example to test PointPillars on KITTI with 8 GPUs and generate a submission t
...
 
-```shell
-mkdir -p results/kitti-3class
+  ```shell
```
Comment (ZCMax) [inline docs/en/advanced_guides/datasets/kitti_det.md:206]: remove this shell code

File: docs/en/advanced_guides/datasets/lyft_det.md [hunk 30]
```diff
@@ -40,8 +40,8 @@ Note that we follow the original folder names for clear organization. Please ren
 
 ## Dataset Preparation
 
-The way to organize Lyft dataset is similar to nuScenes. We also generate the .pkl and .json files which share almost the same structure.
-Next, we will mainly focus on the difference between these two datasets. For a more detailed explanation of the info structure, please refer to [nuScenes tutorial](https://github.com/open-mmlab/mmdetection3d/blob/master/docs/en/datasets/nuscenes_det.md).
+The way to organize Lyft dataset is similar to nuScenes. We also generate the .pkl files which share almost the same structure.
```
Comment (ZCMax) [inline docs/en/advanced_guides/datasets/lyft_det.md:43]: `.pkl`

File: docs/en/advanced_guides/datasets/nuscenes_det.md [hunk 31]
```diff
@@ -26,7 +26,7 @@ mmdetection3d
 
 ## Dataset Preparation
 
-We typically need to organize the useful data information with a .pkl or .json file in a specific style, e.g., coco-style for organizing images and their annotations.
+We typically need to organize the useful data information with a `.pkl` file in a specific style, e.g., coco-style for organizing images and their annotations.
```
Comment (ZCMax) [inline docs/en/advanced_guides/datasets/nuscenes_det.md:29]: remove `, e.g., coco-style for organizing images and their annotations.`

File: docs/en/advanced_guides/datasets/nuscenes_det.md [hunk 32]
```diff
@@ -86,7 +86,7 @@ mmdetection3d
     - info\['instances'\]\['num_lidar_pts'\]: Number of lidar points included in each 3D bounding box.
     - info\['instances'\]\['num_radar_pts'\]: Number of radar points included in each 3D bounding box.
     - info\['instances'\]\['bbox_3d_isvalid'\]: Whether each bounding box is valid. In general, we only take the 3D boxes that include at least one lidar or radar point as valid boxes.
-  - info\['cam_instances'\]: It is a dict contains keys `'CAM_FRONT'`, `'CAM_FRONT_RIGHT'`, `'CAM_FRONT_LEFT'`, `'CAM_BACK'`, `'CAM_BACK_LEFT'`, `'CAM_BACK_RIGHT'`. For vision-based 3D object detection task, we split 3D annotations of the whole scenes according to the camera they belong to.
+  - info\['cam_instances'\]: It is a dict contains keys `'CAM_FRONT'`, `'CAM_FRONT_RIGHT'`, `'CAM_FRONT_LEFT'`, `'CAM_BACK'`, `'CAM_BACK_LEFT'`, `'CAM_BACK_RIGHT'`. For vision-based 3D object detection task, we split 3D annotations of the whole scenes according to the camera they belong to. For the i-th instance:
```
Comment (ZCMax) [inline docs/en/advanced_guides/datasets/nuscenes_det.md:89]: a dict containing

File: docs/en/advanced_guides/datasets/nuscenes_det.md [hunk 33]
```diff
@@ -103,7 +103,7 @@ Note:
 
 2. Here we only explain the data recorded in the training info files. The same applies to validation and testing set (the pkl of test set does not contains `instances` and `cam_instances`).
```
Comment (ZCMax) [inline docs/en/advanced_guides/datasets/nuscenes_det.md:104]: the `.pkl` file of test set xxxx

File: docs/en/advanced_guides/datasets/nuscenes_det.md [hunk 34]
```diff
@@ -86,7 +86,7 @@ mmdetection3d
     - info\['instances'\]\['num_lidar_pts'\]: Number of lidar points included in each 3D bounding box.
```
Comment (ZCMax) [inline docs/en/advanced_guides/datasets/nuscenes_det.md:86]: info\['instances'\]\[i\]\['num_lidar_pts'\]
info\['instances'\]\[i\]\['num_radar_pts'\]
info\['instances'\]\[i\]\['bbox_3d_isvalid'\]

File: docs/zh_cn/advanced_guides/customize_runtime.md [hunk 35]
```diff
@@ -149,19 +137,17 @@ class MyOptimizerWrapperConstructor(DefaultOptimWrapperConstructor):
       _delete_=True, clip_grad=dict(max_norm=35, norm_type=2))
   ```
 
-  如果您的配置继承了一个已经设置了 `optim_wrapper` 的基础配置，那么您可能需要 `_delete_=True` 字段来覆盖基础配置中无用的设置。详见配置文件的[说明文档](https://mmdetection3d.readthedocs.io/en/latest/tutorials/config.html)。
+  如果您的配置继承了一个已经设置了 `optim_wrapper` 的基础配置，那么您可能需要 `_delete_=True` 字段来覆盖基础配置中无用的设置。更多细节请参考[说明文档](https://mmdetection3d.readthedocs.io/en/latest/tutorials/config.html)。
 
-- __使用动量规划器 (momentum scheduler) 来加速模型收敛__：
+- __使用动量调度器器 (momentum scheduler) 来加速模型收敛__：
```
Comment (ZCMax) [inline docs/zh_cn/advanced_guides/customize_runtime.md:142]: 器字重复

**Review by ZCMax — APPROVED** [review]

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**